### PR TITLE
Content translation

### PR DESCRIPTION
**Pull request by Glavo**

### CODE REVIEW

**Comment by wilzbach** [comment]
Hi @Glavo @mogud I have just seen that you both aren't part of the Chinese translation team and thus don't have merge rights ->I have sent out invitations.
Keep on doing this great work!

Btw in case you wonder about:

> object.Exception@source/contentprovider.d(269): Invalid link given: gems/unicode

Within the tour links are checked for correctness. As currently the gem page doesn't exist, you get an error. Two easy solutions:
- Add the missing page as an empty page or (TBD page)
- Remove the link and create an issue, s.t. it's not forgotten

**Comment by mogud** [comment]
@wilzbach Thanks for your advice about CI.
We surely will continue doing this until it's done.

File: basics/memory.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
 
-D provides pointer types `T*` like in C:
+像 C 一样，D 提供指针类型 `T*`：
 
+```d
     int a;
-    int* b = &a; // b contains address of a
-    auto c = &a; // c is int* and contains address of a
+    int* b = &a; // b 存储了 a 的地址
+    auto c = &a; // c 的类型为 int*，并存储了 a 的地址
```
Comment (mogud) [inline basics/memory.md]: 中文好似没有 并 字会好一点

File: basics/memory.md [hunk 2]
```diff
@@ -1,40 +1,40 @@
...
+```d
     void main() @safe {
         int a = 5;
         int* p = &a;
         int* c = p + 5; // error
     }
+```
 
-`@trusted` functions are manually verified functions that allow a bridge between SafeD and the underlying dirty low-level world.
+`@trusted` 函数提供了对代码安全性手动验证的能力，这为 SafeD 与肮脏的低级世界间建立起了一座桥梁。
```
Comment (mogud) [inline basics/memory.md]: 肮脏有点过于贬义，要不要换个词？比如 繁杂 ？

File: basics/memory.md [hunk 3]
```diff
@@ -1,40 +1,40 @@
...
 
-### In-depth
+### 神UR
```
Comment (mogud) [inline basics/memory.md]: 这是什么 0.0 深入么

File: basics/basic-types.md [hunk 4]
```diff
@@ -1,91 +1,70 @@
...
-All data types have a property `.init` to which they are initialized.
-For all integers this is `0` and for floating points it is `nan` (*not a number*).
+所有的数据类型都具有表示该类型初始值的 `.init` 属性，所有整数类型的 `.init` 属性的值都为 `0`，而浮点数为 `nan` (*not a number*).
 
-Integral and floating point types have a `.max` property for the highest value
-they can represent. Integral types also have a `.min` property for the lowest value
-they can represent, whereas floating point types have a `.min_normal` property
-which is defined to the smallest representable normalized value that's not 0.
+整数以及浮点数类型具有表示该类型最大值的 `.max` 属性。
+整数类型也具有表示该类型最小值的 `.min` 属性，而浮点数类型都具有被定义为该类型非0的最小可表示的常规值的 `.min_normal` 属性。
```
Comment (mogud) [inline basics/basic-types.md]: 这后半句感觉可以换下顺序，比如：
而浮点数类型都具有`.min_normal`属性，被定义为该浮点类型非0最小可表示的常规值。

**Pull request merged**